nx-X11/programs/Xserver/hw/nxagent/Font.c

```diff
@@ -73,7 +73,7 @@ is" without express or implied warranty.
 #define NXAGENT_DEFAULT_FONT_DIR      "/usr/share/nx/fonts"
 #define NXAGENT_ALTERNATE_FONT_DIR    "/usr/share/X11/fonts"
 #define NXAGENT_ALTERNATE_FONT_DIR_2  "/usr/share/fonts/X11"
-#define NXAGENT_ALTERNATE_FONT_DIR_3  "/usr/X11R6/lib/X11/fonts"
```

Comment:
This commit was all bogus and sneaked in while rebasing. Commit has been removed.